src/app/api/import-to-sanity/route.ts

```
@@ -45,55 +46,87 @@
-                asset: {
-                  _type: 'reference',
-                  _ref: assetId,
-                },
-                originalUrl: mediaRef.url,
-              }
-            : null
-        })
-        .filter(Boolean) || [],
+// Helper function to find first image asset for cover image
+function findFirstImageAssetId(
+  media: any[],
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)